lms/djangoapps/grades/api/views.py

```diff
@@ -1,24 +1,29 @@
 """ API v0 views. """
 import logging
 
+from django.contrib.auth import get_user_model
 from django.http import Http404
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.keys import CourseKey
 from rest_framework import status
 from rest_framework.authentication import SessionAuthentication
+from edx_rest_framework_extensions.authentication import JwtAuthentication
```

Comment:
FYI: A better alternative is to just update `GradeViewMixin` to use the [`view_auth_classes`](https://github.com/edx/edx-platform/blob/master/openedx/core/lib/api/view_utils.py#L87) decorator.